### ISSUE TITLE
Constructive criticism (MZ)

### ISSUE THREAD

**Issue by jzhu50** [Stanleyhoo1/Indigo__stanleyh28_colyic_jacobl153_naomil49#3]
Grow: Maybe have different titles for different pages. For example, “Stories in progress” for the page with the stories in progress instead of “Home”.
Question: what does “{{ content | safe }}” do?

**Comment by Stanleyhoo1** [comment]
Yeah, I forgot to change the titles for those pages since they were just copy pastes of the homepage. "{{ content | safe }}" basically tells the html that the variable you are inserting can be treated as html code rather than text as is the case when inserting regular variables. For example, when passing in the argument for render_template I said render_template('home.html', content=&lt;h1&gt;Hello&lt;/h1&gt;) when displayed on the page using {{ content }} it will print &lt;h1&gt;Hello&lt;/h1&gt; but if you use {{ content | safe }} it will display Hello in the h1 font
